app/start.py

```diff
@@ -14,13 +14,12 @@
 # limitations under the License.
 
 import const
-from utils import *
-from model import *
+from utils import BaseHandler
+from model import Counter
 
 
 class Handler(BaseHandler):
     repo_required = False
-
```

Comment:
Do not remove this blank line. There should be a blank line between member definitions in a class.

Reply:
done.